### ISSUE TITLE
How to pass `from`option?

### ISSUE THREAD

**Issue by buz-zard** [postcss/postcss-js#41]
Hello, I'm getting warnings - 
```Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning.```

It seems that you can't pass the `from` option, or it should bet set to `undefined` in the first place since this is supposed to be used only in the browser .

**Comment by ai** [comment]
How do you use PostCSS?

`postcss-js` shouldn’t show this warning since it doesn’t use `postcss.parse()`

**Comment by buz-zard** [comment]
It seems that only the `async` method warns about it:

```js
import postcssJs from "postcss-js";

postcssJs.async([])({ padding: "10px" });
```

Here's the demo - https://codesandbox.io/s/dreamy-jennings-xqe7t

**Comment by ai** [comment]
Got it. Fixed 197c5f3

**Comment by ai** [comment]
Released in 2.0.3
